Makefile/Makefile.sublime-settings

```diff
@@ -1,4 +0,0 @@
-{
-	"translate_tabs_to_spaces": false,
```

Comment:
I suggest setting `translate_tabs_to_spaces` to `false` while still having `detect_indentation` as `true`. It will use the sane default while still being able to detect files that are indented with a single space (or any other character), in case anyone does use `.RECIPEPREFIX`.